viz_scripts/plots.py

```diff
@@ -369,6 +349,38 @@ def store_alt_text_timeseries(df, chart_name, var_name):
     alt_text = access_alt_text(alt_text, chart_name)
     return alt_text
 
+# Creating html table with col as Trip Type, Mode, Count, and Proportion
+def access_alt_html(alt_text, chart_name, var_name):
```

Comment:
why is the first argument to this `alt_text` when it is printing out HTML?
Also, why do we need a copy-pasted function that essentially has only a few lines of code?

Reply:
Updated the first argument to `alt_html`